doc/editorconfig.txt

```diff
@@ -124,25 +124,29 @@ is restarted.
 
                                             *g:EditorConfig_max_line_indicator*
 The way to show the line where the maximal length is reached. Accepted values
-are "line", "fill" and "exceeding", otherwise there will be no max line
-indicator.
+are "line", "fill", "exceeding" and "fillexceeding", otherwise there will be
+no max line indicator.
 
-    "line":      the right column of the max line length column will be
-                 highlighted, made possible by adding "+1" to 'colorcolumn'.
+    "line":           the right column of the max line length column will be
+                      highlighted, by adding "+1" to 'colorcolumn'.
```

Comment:
liplease change to "highlighted on all lines" to to make the difference between `line` and `exceeding` more clear (it was not at all apparent to me until I tried all four values just now :) ).

Reply:
Updated to "highlighted on all lines", separate commit as not directly related to new `fillexceeding` option